functions/Get-DbaRegisteredServer.ps1

```diff
@@ -0,0 +1,159 @@
+function Get-DbaRegisteredServer {
+	<#
+		.SYNOPSIS
+			Gets list of SQL Server objects stored in SQL Server Central Management Server (CMS).
+
+		.DESCRIPTION
+			Returns an array of servers found in the CMS.
+
+		.PARAMETER SQLInstance
+			SQL Server instance(s) to connect to.
```

Comment:
Please use the instance help defs that andy's been using - they can also be found in the template

This should also be SqlInstance and not SQLInstance

Reply:
I copied the style changes that he made in 4dcab49e5e0cc2e0dabcd820c2d7ad1446c6dc8a ???

Reply:
mmm that commit has SqlInstance and not SQLInstnace 

https://github.com/sqlcollaborative/dbatools/blob/4dcab49e5e0cc2e0dabcd820c2d7ad1446c6dc8a/functions/Export-DbaLogin.ps1